### PR TITLE
Feature/autosuggest category

### PR DESCRIPTION
**Pull request by royderks**

### CODE REVIEW

File: src/app/components/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+const StyledSearchBar = styled(SearchBarComponent)`
+  width: 100%;
+`
+
+type SearchBarProps = {
+  expanded: boolean
+  onOpenSearchBarToggle: Function
+  openSearchBarToggle: boolean
+  value: string
+  onBlur: () => void
```
Comment (chrisvanmook) [inline src/app/components/SearchBar/SearchBar.tsx:44]: I think if you unify these props with `React.InputHTMLAttributes<HTMLInputElement>` you don't need to specify these props

File: src/header/components/auto-suggest/AutoSuggest.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  index: number
+  label: string
+  uri: string
+  type: CmsType | SearchType
+}
+
+export type SuggestionList = Array<{
+  label: string
+  // eslint-disable-next-line camelcase
+  total_results: number
```
Comment (chrisvanmook) [inline src/header/components/auto-suggest/AutoSuggest.tsx:17]: can't we make this camelCase?

File: package.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
     "@datapunt/arm-draw": "^0.2.8-alpha.63",
     "@datapunt/arm-nontiled": "^0.2.8-alpha.49",
     "@datapunt/asc-assets": "^0.20.0",
-    "@datapunt/asc-ui": "^0.21.1",
+    "@datapunt/asc-ui": "^0.20.1-alpha.13",
```
[the remark this answers is not in the document]
Comment (royderks) [reply package.json:42]: The local versioning of asc-ui updated to this version when I ran a canary release...

[the remark this answers is not in the document]
Comment (chrisvanmook) [reply package.json:42]: that's odd, could you look if you can release a canary with a higher version than 0.21.1?

**Pull request merged**